### ISSUE TITLE
Move to DB configs

### ISSUE THREAD

**Issue by jlingema** [jlingema/cmssw#4]
To make configuring from DB possible need to implement an ESProducer and CondFormat object.

Calo ESProducer:
https://github.com/cms-sw/cmssw/blob/CMSSW_7_6_X/L1Trigger/L1TCalorimeter/plugins/L1TCaloParamsESProducer.cc
Calo CondFormat:
https://github.com/cms-sw/cmssw/blob/CMSSW_7_6_X/CondFormats/L1TObjects/interface/CaloParams.h

**Comment by thomreis** [comment]
I have started to work on this in branch
https://github.com/thomreis/cmssw/tree/l1t-muon-CMSSW_7_6_0_pre4_microGMT_paramsESProducer

Still has two compile errors that I need to resolve.

**Comment by thomreis** [comment]
Moved to CMSSW_7_6_0_pre5 in new branch
https://github.com/thomreis/cmssw/tree/l1t-muon-CMSSW_7_6_0_pre5_microGMT_paramsESProducer

The ESProducer puts the firmware version and the paths to the LUTs in the ES.
I have adapted the emulator (now named L1TMicroGMTProducer) to read these from the ES and generate the LUTs from it.
Factories for the LUTs exist now that select the correct implementation according to the FW version.
With this the input and output widths for the LUTs will be hardcoded in the concrete implementation for a FW version and dropped from the configuration.
